### PR TITLE
[Documentation] Document how to test a Chart in the Review Guidelines.

### PR DESCRIPTION
**Pull request by desaintmartin**
cc @helm/charts-maintainers

### CODE REVIEW

**Comment by desaintmartin** [comment]
Hm, I can't ping all charts maintainers this way.
Let's do this the old way.
@lachie83 @mgoodness @prydonius @sameersbn @seanknox @viglesiasce @foxish @unguiculus @scottrigby @mattfarina @davidkarlsen @paulczar @cpanato

**Comment by desaintmartin** [comment]
Now, yes. :)
I'm going to rewrite. Thanks!

File: REVIEW_GUIDELINES.md
Comment (prydonius) [inline REVIEW_GUIDELINES.md]:
```suggestion
The `ci` directory of a given Chart allows testing different use cases, by allowing you to define different sets of values overriding `values.yaml`, one file per set. See the [documentation](https://github.com/helm/charts/blob/master/test/README.md#providing-custom-test-values) for more information.
```

File: test/README.md
Comment (prydonius) [inline test/README.md]: allows deploying a release for the changed Helm Chart to test it?

File: test/README.md
Comment (prydonius) [inline test/README.md]: `e.g.` instead of `examples`?

File: test/README.md
Comment (prydonius) [inline test/README.md]: A better way to put this might be:

Please note that in order to test using the default values when using the `ci` directory, an empty values file must be present in the directory.

File: test/README.md
Comment (prydonius) [inline test/README.md]: For examples, you can take a look at existing tests in this repository (e.g. [Kibana Chart](https://github.com/helm/charts/tree/master/stable/kibana/ci)). Also, we should point to the current commit rather than master in case that chart changes.

File: test/README.md
Comment (prydonius) [inline test/README.md]: Aren't they related, in that a Helm Chart Test will get run after deploying the release?

Comment (desaintmartin) [reply test/README.md]: Yes, let's explain this.

File: REVIEW_GUIDELINES.md
Comment (prydonius) [reply REVIEW_GUIDELINES.md]: hmm did you update this one? Not seeing the changes

**Review by prydonius — APPROVED** [review]
thanks a lot for this @desaintmartin, it lgtm but will wait for someone else to also lgtm

File: REVIEW_GUIDELINES.md
Comment (desaintmartin) [reply REVIEW_GUIDELINES.md]: Right, the commit has been overridden.

**Comment by desaintmartin** [comment]
Up

This one is small but enforces QA as a good practice.

**Comment by viglesiasce** [comment]
/lgtm

Thanks @desaintmartin!!

**Pull request merged**
